CONTRIBUTING.md

```diff
@@ -161,6 +161,20 @@ Cluster API maintains the most recent release/releases for all supported API and
 
 - Exceptions can be filed with maintainers and taken into consideration on a case-by-case basis.
 
+### Removal of v1alpha3 & v1alpha4 apiVersions
+
+The last releases supporting v1alpha3 (v0.3.x) and v1alpha4 (v0.4.x) apiVersions have been both EOL since April 2022.
+The apiVersions have been kept around to allow upgrades from v0.3.x and v0.4.x Cluster API versions to v1.x. Given that 
+these releases have been EOL for a significant period, we are going to remove the apiVersions in upcoming releases:
```

Comment:
```suggestion
We are going to remove the apiVersions in upcoming releases:
```
I think we should keep it simple and straight to the point the readers should care about, but no objection if we prefer the long version